context/php/disabled/newrelic.ini

```diff
@@ -4,6 +4,7 @@ newrelic.license = ${NEWRELIC_LICENSE}
 newrelic.appname = ${NEWRELIC_APPNAME}
 newrelic.framework = "no_framework"
 newrelic.browser_monitoring.auto_instrument = false
+newrelic.daemon.location = "/usr/bin/newrelic-daemon"
```

Comment:
Why do we need to introduce this options?

Reply:
Latest Agent doesn't work without daemon